### DISCUSSION TITLE
V 1.0 on Docker Synology

### DISCUSSION THREAD

**Discussion by agentdogit** [itzg/docker-minecraft-server#1319] [Q&A]
Hi there

I want to run version 1.0 version of Minecraft for my son, which version do I need to select from the list as there are many? I can see 2021 1.0 would that be the first version of MC?

**Comment by itzg** [comment]
The Minecraft version is completely independent of the image version. I recommend always using the latest image version and then refer to

https://github.com/itzg/docker-minecraft-server#versions

**Reply by agentdogit** [reply]
yes I saw this but I do not understand the versions relating to the Java versions, which image version would I need to run version 1.0?

**Reply by itzg** [reply]
For the image use the latest of `itzg/minecraft-server:java8-multiarch`. Please note that the oldest server version Mojang provides is 1.2.1, not 1.0.
